.travis.yml

```diff
@@ -32,10 +33,16 @@ addons:
   apt:
     packages:
       - swig
+  firefox: latest
```

Comment:
@jrbenny35 'stable' would be a great alias for this version - maybe we could patch the Travis CI addon to support this?